Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM ubuntu:bionic
+FROM ubuntu:focal
 
 # Install packages required to build AsteroidOS
 # And add the en_US.utf8 locale because it is required and not installed by default in minimal Ubuntu images
-RUN apt update && apt upgrade -y && apt install -y git build-essential cpio diffstat gawk chrpath texinfo python python3 python3-distutils wget shared-mime-info zstd liblz4-tool locales \
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
```

Comment:
Although the TZ variable isn't set under Ubuntu, the time zone is already UTC as can be verified by running `date`.  Is there some reason this is necessary?

Reply:
When I was testing that it interactively asked to select time zone because it wasn't set previously. I am not sure if is this best practice, it simply worked for me.

Reply:
It seems to work for me without it.  Can you please verify that's the case for you as well?

Reply:
I did it together with `ENV DEBIAN_FRONTEND=noninteractive`. I took it from other project, but it turned out it works also for me without it so I have removed it.